src/enhancers/mod.rs

```diff
@@ -0,0 +1,5 @@
+//! This module contains various functions that enhance / cleanup `Collections`
+
+mod enhance_co2;
+
+pub(crate) use enhance_co2::enhance_with_co2;
```

Comment:
Since it's part of the module `enhancers`, maybe the function can be just named `with_co2()` now? And you would use it as `enhancer::with_co2()`? What do you think?

Reply:
I'm not convinced by the `enhancer` name so I don't know :grin:
I plan to also move the `sanitize` method into it so it would be `enhancer::sanitize`? (well, having written it I don't really think it's a problem, what do you think?)

Reply:
Maybe we can find a better name for the module? `filters`? `refine`? `process`? Since this module is part of the `model` module, I believe, it's clear that it is applied on `Model`.

I kind of like `process` or even `refine`...

Reply:
Not fan of 'process' (too blurry) or 'filter' (there is more than that into this), but 'refine' or 'enhance' is OK to me.

Reply:
so `refine::enhance_with_co2` and `refine::sanitize`? not mind blown but fine for me

Reply:
Well that would be `refine::with_co2` but really, `enhancer`, `enhance`, `refiner` or `refine` are all good for me, my first comment was more on removing `enhance_` from the name of the function since we now have the module's name for that...

Reply:
Then `fill_co2`?

Reply:
`enhancers::fill_co2` seems good for me